pkg/client/message.go

```diff
@@ -17,8 +17,8 @@ package client
 
 // Message is an implementation of Message interface
 type Message struct {
-	contentType  string
-	messageBody  string
-	messageCount int
-	messageID    string
+	ContentType  string
+	MessageBody  string
+	MessageCount int
+	MessageID    string
```

Comment:
These should be `Body`, `Count` and `ID`, as `message` is implicit. Users will probably write something like this:

```go
message.MessageBody := "Hello!"
```

But it would be better to have this:

```go
message.Body := "Hello!"
```

We should also consider making this an interface, and providing only read methods, to make messages immutable.

Reply:
:+1:  On it :exclamation:

Reply:
done in https://github.com/container-mgmt/messaging-library/pull/12